main/iam-policy-sqs.tf

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_iam_policy" "beekeeper_sqs" {
```

Comment:
I personally like to have policy, role and attachment together in one file `IAM-sqs.tf` but if this is the pattern we are following in this project, I am fine with it.